planner/cascades/testdata/integration_suite_out.json

```diff
@@ -406,6 +406,71 @@
       }
     ]
   },
+  {
+    "Name": "TestDoubleRead",
+    "Cases": [
+      {
+        "SQL": "select a, b from t order by t.a",
+        "Plan": [
+          "Projection_18 10000.00 root test.t.a, test.t.b",
+          "└─IndexLookUp_17 10000.00 root ",
+          "  ├─IndexScan_19 10000.00 cop[tikv] table:t, index:a, range:[NULL,+inf], keep order:true, stats:pseudo",
+          "  └─TableScan_20 10000.00 cop[tikv] table:t, range:[-inf,+inf], keep order:true, stats:pseudo"
+        ],
+        "Result": [
+          "1 11",
+          "2 22",
+          "3 33",
+          "4 44"
+        ]
+      },
+      {
+        "SQL": "select a, b from t order by t.b",
+        "Plan": [
+          "Projection_18 10000.00 root test.t.a, test.t.b",
+          "└─IndexLookUp_17 10000.00 root ",
+          "  ├─IndexScan_19 10000.00 cop[tikv] table:t, index:b, c, range:[NULL,+inf], keep order:true, stats:pseudo",
+          "  └─TableScan_20 10000.00 cop[tikv] table:t, range:[-inf,+inf], keep order:true, stats:pseudo"
+        ],
+        "Result": [
+          "1 11",
+          "2 22",
+          "3 33",
+          "4 44"
+        ]
+      },
+      {
+        "SQL": "select a, b, c from t order by t.b, t.c",
+        "Plan": [
+          "Projection_18 10000.00 root test.t.a, test.t.b, test.t.c",
+          "└─IndexLookUp_17 10000.00 root ",
+          "  ├─IndexScan_19 10000.00 cop[tikv] table:t, index:b, c, range:[NULL,+inf], keep order:true, stats:pseudo",
+          "  └─TableScan_20 10000.00 cop[tikv] table:t, range:[-inf,+inf], keep order:true, stats:pseudo"
```

Comment:
`TableScan` does not need to keep order. Only `IndexScan` can keep order here.
For example, in old planner:
```
mysql> desc select * from tt where b > 10 order by b;
+----------------------+---------+-----------+-------------------------------------------------------------------+
| id                   | count   | task      | operator info                                                     |
+----------------------+---------+-----------+-------------------------------------------------------------------+
| Projection_18        | 3333.33 | root      | test.tt.a, test.tt.b, test.tt.c, test.tt.d                        |
| └─IndexLookUp_17     | 3333.33 | root      |                                                                   |
|   ├─IndexScan_15     | 3333.33 | cop[tikv] | table:tt, index:b, range:(10,+inf], keep order:true, stats:pseudo |
|   └─TableScan_16     | 3333.33 | cop[tikv] | table:tt, keep order:false, stats:pseudo                          |
+----------------------+---------+-----------+-------------------------------------------------------------------+
4 rows in set (0.00 sec)
```